docs/user/guides/setup/git-repo-automatic.rst

```diff
@@ -1,27 +1,54 @@
-How to connect your Git repository
-==================================
+How to automatically configure a Git repository
+===============================================
 
 In this article, we explain how connecting your Read the Docs account to |git_providers_or|
-automatically configures your Git repository and your Read the Docs project.
+makes Read the Docs able to automatically configure your imported Git repositories and your Read the Docs projects.
 
 ✅️ Signed up with your Git provider?
   If you signed up or logged in to Read the Docs with your |git_providers_or|
   credentials, you're all done. Your account is connected.
-  The permissions that are granted are :ref:`explained below <connected-accounts:Permissions for connected accounts>`.
-⬇️ Signed up with your email address?
+
+  The rest of this guide helps to understand how the automatic configuration works.
+
+⏩️️ Signed up with your email address?
   If you have signed up to Read the Docs with your email address,
   you can add the connection to the Git provider afterwards.
-  See: :doc:`/guides/connecting-git-account`
+  You can also add a connection to an additional Git provider this way.
+
+  Please follow :doc:`/guides/connecting-git-account` in this case.
+
+Automatic configuration
+-----------------------
+
+When your Read the Docs account is connected to your Git Provider and you import a Git repository,
+all the basic settings will automatically be configured on the Read the Docs project and on your Git repository.
+
+Here is an outline of what happens:
+
+#. A list of repositories that you have access to are automatically listed on Read the Docs' project import.
+#. You choose a Git repository from the list (see :doc:`/intro/import-guide`).
+#. Data about the repository is now read using the account connection and you are asked to confirm the setup.
+#. Read the Docs creates the project with these basic settings,
+   including a unique webhook that the Git provider calls.
+#. Read the Docs uses the account connection to add its webhook URL on the Git repository's settings,
+   as well as adding an integration for :doc:`Pull Request builds </guides/pull-requests>`.
```

Comment:
Does it add this? You need to enable PR builds in the project settings, I don't think it's enabled by default?

Reply:
Ah yes, that's definitely good to clarify

> #. Read the Docs also configures your project to use the Git provider's webhook via your account connection,
>   so your project is ready to enable :doc:`Pull Request builds </guides/pull-requests>`.

The point here is to illustrate that the project is automatically being setup to use the account connection of the one that sets it up.

This will improve a lot with GitHub Apps.